async.h

```diff
@@ -36,7 +36,8 @@
  * NOTE: all values in this enum must be negative
  */
 enum async_ret_code {
-	ASYNC_NO_IO = -5,
+	ASYNC_NO_IO = -6,
+	ASYNC_CHANGE_FD,
```

Comment:
I'd suggest moving `ASYNC_CHANGE_FD` two spots below, since it's still an async operation (but on a new DB backend/host), along with `ASYNC_CONTINUE`.